src/main/java/com/novauc/MessageController.java

```diff
@@ -0,0 +1,45 @@
+package com.novauc;
+
+import com.sun.javafx.sg.prism.NGShape;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by jerieshasmith on 3/15/17.
+ */
+@Controller
+public class MessageController {
+
+    @Autowired
+    MessageRepository messages;
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model) {
+        List<Message> messageList = (ArrayList) messages.findAll();
+        model.addAttribute("messages", messageList);
+        return "home";
+
+
+    }
+
+    @RequestMapping(path = "/add-message", method = RequestMethod.POST)
+    public String addMessage(String messageText) {
+        Message message = new Message(messageText);
+        messages.save(message);
+        return "redirect:/";
+    }
+        @RequestMapping(path ="/edit-message", method =RequestMethod.POST)
+        public String editMessage(String editmessage){
+        Message m = new Message() ;
+        messages.findOne(1);
+        messages.save(m);
+        return "redirect:/";
```

Comment:
Edit doesn't work.  messages.findOne is the right method but you are putting the wrong thing into it.  And then you are not SETting the message to any new information, before you save.  Fix this and resubmit.